### PR TITLE
Adds UHC Behavioral Health Variables

### PR DESCRIPTION
**Pull request by benhammondmusic**
closing; this PR was split into #1355  and #1353

### CODE REVIEW

**Comment by benhammondmusic** [comment]
Screen Shot 2021-12-04 at 2 43 17 PM
Screen Shot 2021-12-04 at 2 43 24 PM

**Comment by benhammondmusic** [comment]
Screen Shot 2021-12-09 at 2 16 10 PM
Screen Shot 2021-12-09 at 2 16 29 PM
Screen Shot 2021-12-09 at 2 16 47 PM
Screen Shot 2021-12-09 at 2 17 21 PM

File: python/datasources/acs_population.py [hunk 1]
```diff
@@ -104,15 +104,39 @@ def get_decade_age_bucket(age_range):
...
+                       '25-29', '30-34', '35-44', '35-39', '40-44'}:
         return '18-44'
     elif age_range in {'45-54', '45-49', '50-54', '55-64', '55-59', '60-61', '62-64'}:
         return '45-64'
     elif age_range in {'65-74', '65-66', '67-69', '70-74', '75-84', '75-79', '80-84', '85+'}:
         return '65+'
-    elif age_range == 'Total':
+
+
+def get_uhc_age_bucket_more(age_range):
```
Comment (joshzarrabi) [inline python/datasources/acs_population.py:120]: can you think of more descriptive names for these?

File: README.md [hunk 2]
```diff
@@ -290,52 +290,52 @@ Before deploying, make sure you have installed Terraform and a Docker client (e.
 * Login to glcoud
 
 ```bash
-$ gcloud auth application-default login
```
Comment (joshzarrabi) [inline README.md:293]: I'm fine with these changes but can you do them in another pr

Comment (benhammondmusic) [reply README.md:293]: moved to #1352
